src/qweb/extensions.ts

```diff
@@ -77,7 +77,7 @@ export function makeHandlerCode(
     code = ctx.captureExpression(value);
   }
   const modCode = mods.map((mod) => modcodes[mod]).join("");
-  let handler = `function (e) {if (context.__owl__.status === ${STATUS.DESTROYED}){return}${modCode}${code}}`;
+  let handler = `function (e) {if (context.__owl__.status === ${STATUS.DESTROYED}){return}${modCode}const res = (() => { return ${code} })(); if (typeof res === 'function') { res(e) }}`;
```

Comment:
I couldn't figure out how to store the result of `${code}` without wrapping it in a function.